bin/healthchecks/collectorredis/metrics

```diff
@@ -8,5 +8,6 @@
 #
 ##############################################################################
 
-/opt/zenoss/bin/redis-mon post
-/opt/zenoss/bin/redis-mon post-length --list metrics
+(timeout 10s /opt/zenoss/bin/redis-mon post) &
+(timeout 10s /opt/zenoss/bin/redis-mon post-length --list metrics) &
+redis-cli ping
```

Comment:
That could happen if the timeouts here are longer than the healthcheck interval. I haven't seen the process count get out of hand in my testing.